### DISCUSSION TITLE
Monte Carlo Stochastic Reserving

### DISCUSSION THREAD

**Discussion by Vincent-EDSA** [casact/chainladder-python#326] [Ideas]
The current Bootstrapped stochastic reserving method is sufficient for reserve confidence interval estimation however, an additional stochastic model such as Monte Carlo Simulation might be a good way to compare/ validate the process errors as calculated with the Mack and Bootstrapped methods.

**Comment by jbogaardt** [comment]
Hi @Vincent-EDSA , sorry for the late reply. Can you elaborate on the idea? Is there a public paper/example that we can reference?

**Reply by phuang1226** [reply]
Hi @jbogaardt ,  there is a CAS monograph by Glenn Meyers:  https://www.casact.org/sites/default/files/2021-02/01-Meyers.PDF

Thanks.

**Reply by jbogaardt** [reply]
I see, I believe we have an open issue (#30) open for this monograph. My experience with MCMC is limited and I think we might need an extra dependency on `pymc` to accomplish this.

**Comment by Vincent-EDSA** [comment]
Hi @jbogaardt, the attached paper might shed some light on the MC-Chain Ladder. At a high level, here we rather sample the age-age development factors - volume-weighted to ensure the correct distribution results.

[MonteCarloBasicChainLaddrTheory.pdf](https://github.com/casact/chainladder-python/files/9852571/MonteCarloBasicChainLaddrTheory.pdf)
